### PR TITLE
x11-terms/eterm: EAPI 3 -> 6

### PR DESCRIPTION
**Pull request by Moiman**
Package-Manager: Portage-2.3.16, Repoman-2.3.6

### CODE REVIEW

File: x11-terms/eterm/eterm-0.9.6-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		$(use_enable utempter utmp) \
+		--with-delete=execute \
+		--with-backspace=auto
+}
+
+src_install() {
+	use escreen && DOCS+=( doc/README.Escreen )
+	default
+	# We don't install headers to link against this library
+	rm -f "${D}"/usr/*/libEterm.{so,la}
```
Comment (Amynka) [inline x11-terms/eterm/eterm-0.9.6-r1.ebuild]: Missing || die

File: x11-terms/eterm/eterm-0.9.6-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_P=Eterm-${PV}
```
Comment (SoapGentoo) [inline x11-terms/eterm/eterm-0.9.6-r1.ebuild]: instead of `${MY_P}`, use `${P^}`

File: x11-terms/eterm/eterm-0.9.6-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+SRC_URI="http://www.eterm.org/download/${MY_P}.tar.gz
+		!minimal? ( http://www.eterm.org/download/Eterm-bg-${PV}.tar.gz )"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~ppc-macos ~x86-macos"
+
+DESCRIPTION="A vt102 terminal emulator for X"
```
Comment (SoapGentoo) [inline x11-terms/eterm/eterm-0.9.6-r1.ebuild]: `DESCRIPTION`, `HOMEPAGE`, `SRC_URI`, free line, `LICENSE`, `SLOT`, `KEYWORDS`, `IUSE`

File: x11-terms/eterm/eterm-0.9.6-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+		$(use_enable utempter utmp) \
+		--with-delete=execute \
+		--with-backspace=auto
+}
+
+src_install() {
+	use escreen && DOCS+=( doc/README.Escreen )
+	default
+	# We don't install headers to link against this library
+	rm -f "${D}"/usr/*/libEterm.{so,la} || die
```
Comment (SoapGentoo) [inline x11-terms/eterm/eterm-0.9.6-r1.ebuild]: `rm -f "${ED%/}"/usr/*/libEterm.{so,la} || die`

**Pull request closed without merging**